ui/index.html

```diff
@@ -1,39 +1,61 @@
 <!DOCTYPE html>
 <html>
-   <head>
-      <script src="jquery.js"></script>
-      <script src="rla_ui.js"></script>
-      <link rel="stylesheet" href="style.css">
-   </head>
-   <body>
-      <div id="auditTypeContainer" class="container">
-         Choose the audit type:<br />
-      </div>
-      <div id="auditNameContainer" class="container" style="display:none">
-         Enter a name for this audit:<br />
-         <input type="text" id="auditNameBox" />
-         <button id="auditNameSaveButton">Save</button>
-      </div>
-      <div id="cvrFileUploadContainer" class="container" style="display:none">
-         <p>Upload a Cast Vote Record (CVR) file</p>
-         <form id="uploadCVRForm" method="post" enctype="multipart/form-data">
+
+<head>
```

Comment:
(Basically the opposite of my advice for the JS file, below:)

Since the file started out with indentation before e.g. `<head>` and `<body>`, it's probably best to leave it that way to reduce the diff noise.

Reply:
(Opposite indentation advice, but both in the service of making the smallest possible diff)